os/net/lwip/src/core/ipv4/inet_chksum.c

```diff
@@ -57,7 +57,7 @@
 #include <net/lwip/debug.h>
 #include <net/lwip/ipv4/inet_chksum.h>
 #include <net/lwip/def.h>
-
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting 'htons' related error to overcome that error we have added this header file.